Sources/BraintreeDropIn/BTPaymentSelectionViewController.m

```diff
@@ -476,7 +471,6 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
         }
     } else if (cell.type == BTUIKPaymentOptionTypePayPal) {
         BTPayPalDriver *driver = [[BTPayPalDriver alloc] initWithAPIClient:self.apiClient];
-        driver.viewControllerPresentingDelegate = self.delegate;
```

Comment:
As of v5, there's no more `viewControllerPresentingDelegate` on `BTPayPalDriver`.